nebari-dev/nebari
# Add taint to user and worker nodes

## Reference Issues or PRs
Fixes https://github.com/nebari-dev/nebari/issues/2507

- [x] I need to test running pods with Argo Workflow through Nebari Workflow Controller before merging this PR



## What does this implement/fix?

_Put a `x` in the boxes that apply_


## Testing


### How to test this PR
A few possible ways to test:
- You could not specify taints, do a deployment and then add taints, redeploy and make sure things are still working.
- You could do a deployment and specify taints and make sure things work.
- You could do a local deployment and make sure it deploys and make sure things work.

Things working is defined as:
- jupyterlab server spins up for user
- dask scheduler and worker spins up for user (when using nebari-git-dask built in env).  Use some code like [this](https://github.com/nebari-dev/nebari-demo/blob/main/04-dask.ipynb)


## Any other comments?
